### DISCUSSION TITLE
Enable secondary bg color of objects

### DISCUSSION THREAD

**Discussion by SaucySauce** [uxbox/uxbox#628] [Ideas]
![Screenshot from 2021-02-10 19-42-06](https://user-images.githubusercontent.com/38703177/107569689-23a42500-6bd8-11eb-93d1-52dbb74678b5.png)

Right now the only fix would be to add an identical object in the background with the secondary or base color.

**Comment by myfunnyandy** [comment]
Hi @SaucySauce, we've been discussing the convenience of allowing more than a fill to shapes and we decided to add it as an enchancement. You can se it here: https://tree.taiga.io/project/penpot/issue/1248

Thanks for the suggestion!
